R/bin.R

```diff
@@ -87,6 +87,10 @@ bin_breaks_width <- function(x_range, width = NULL, center = NULL,
   max_x <- x_range[2] + (1 - 1e-08) * width
   breaks <- seq(origin, max_x, width)
 
+  if (length(breaks) > 1e6) {
+    stop("Number of bins exceeds the maximum of 1e6", call. = FALSE)
```

Comment:
Can you write in the style of http://style.tidyverse.org/error-messages.html? I think a hint of the form "Did you make `binwidth` too small?` would be useful.

Reply:
Ah - sure